test/fixtures/files/fairdatastation/demo.ttl

```diff
@@ -0,0 +1,877 @@
+@prefix base:     <http://fairbydesign.nl/ontology> .
+@prefix bibo:     <http://purl.org/ontology/bibo/> .
+@prefix dc:       <http://purl.org/dc/terms/> .
+@prefix dcmitype: <http://purl.org/dc/dcmitype/> .
+@prefix empusa:   <http://empusa.org/0.1/> .
+@prefix fair:     <http://fairbydesign.nl/ontology/> .
+@prefix foaf:     <http://xmlns.com/foaf/0.1/> .
+@prefix gbol:     <http://gbol.life/0.1/> .
+@prefix jerm:     <http://jermontology.org/ontology/JERMOntology#> .
+@prefix mixs:     <https://w3id.org/gensc.org/terms/> .
+@prefix owl:      <http://www.w3.org/2002/07/owl#> .
+@prefix ppeo:     <http://purl.org/ppeo/PPEO.owl#> .
+@prefix prov:     <http://www.w3.org/ns/prov#> .
+@prefix rdf:      <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs:     <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sample:   <http://fairbydesign.nl/sample/> .
+@prefix schema:   <http://schema.org/> .
+@prefix sschema:  <https://schema.org/> .
+@prefix uniprot:  <http://purl.uniprot.org/core/> .
+@prefix unlock:   <http://m-unlock.nl/ontology/> .
+@prefix void:     <http://m-unlock.nl/ontology/> .
+@prefix wd:       <http://www.wikidata.org/wiki/> .
+@prefix wdt:      <http://www.wikidata.org/wiki/Property:> .
+@prefix wv:       <http://vocab.org/waiver/terms/norms> .
+@prefix xml:      <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd:      <http://www.w3.org/2001/XMLSchema#> .
+
+fair:inv_INV_DRP007092
+        rdf:type            jerm:Investigation ;
+        jerm:hasPart        <http://fairbydesign.nl/ontology/inv_INV_DRP007092/stu_DRP007092> ;
+        schema:contributor  <http://fairbydesign.nl/ontology/inv_INV_DRP007092/person/343732a9426f424e535356c966739de9> ;
+        schema:description  "Exploration of HIV-1 infected individuals in Ghana" ;
+        schema:identifier   "INV_DRP007092" ;
+        schema:title        "HIV-1 infected individuals in Ghana" .
+
+<http://fairbydesign.nl/ontology/inv_INV_DRP007092/person/343732a9426f424e535356c966739de9>
+        rdf:type           schema:Person ;
+        schema:department  "AIDS Research Center" ;
+        schema:email       <mailto:pkofieparbie@nih.go.jp> ;
```

Comment:
Maybe worth replacing with a dummy email address

Reply:
it's a public dataset I checked could be shared, but I've replaced it anyway.